docs/api-reference/sql-jdbc.md

```diff
@@ -30,7 +30,8 @@ sidebar_label: SQL JDBC driver
 
 
 You can make [Druid SQL](../querying/sql.md) queries using the [Avatica JDBC driver](https://calcite.apache.org/avatica/downloads/).
-We recommend using Avatica JDBC driver version 1.22.0 or later.
+We recommend using Avatica JDBC driver version 1.22.0 or later. Note that starting with Avatica 1.21.0, you may need to set the [`transparent_reconnection`](https://calcite.apache.org/avatica/docs/client_reference.html#transparent_reconnection) property to `true` if you notice intermittent query failures.
```

Comment:
Since we are using 1.23 now should this be 1.23 ? Or since the change came after 1.21 this should be 1.21 ?

Reply:
I don't know where that original recommendation for the version came from, so your suggestion for versions would probably be what we go with.

Reply:
Ok let's change this to 1.23. Rest looks good

Reply:
Ok lets change this to 1.23 rest looks good